### PR TITLE
Initialize Felt project roadmap/timeline

### PR DESCRIPTION
**Pull request by greatbacon**
This PR initializes a `roadmap` (although currently it's mostly just a historical timeline) on our website. Way back in July we had chatted a little bit about this and I've had it on my backburner since. Currently it is unlisted like the blog.

I think I've got a decent first pass on styling, but I wanted to run it past you.

The actual content is also not fully filled out, nor is any of it final atm. Any feedback appreciated on what's there. 

I was thinking we can hammer out the structure/style and the current content in this PR, and then open a followup PR to finish fleshing out content & link to it.

### CODE REVIEW

File: src/lib/roadmap/roadmapData.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+export interface RoadmapItems {
+	icon: string;
+	status: string;
```
Comment (ryanatkn) [inline src/lib/roadmap/roadmapData.ts:3]:
```suggestion
	status: 'completed' | 'in-progress' | 'completed' | 'milestone';
```

**Comment by ryanatkn** [comment]
can you think of a representative emoji for the roadmap for the front page?

File: src/lib/roadmap/roadmapData.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+			'Published this piece (https://github.com/feltcoop/why-svelte) about why we are using Svelte.',
+		date: 'Q1 2020',
+	},
+	{
+		icon: '🌰',
+		status: 'completed',
+		title: 'First commit to our build tool, Gro',
+		summary: 'Work begins in earnest on our builder & taskrunner, Gro.',
+		date: 'Q2 2020',
+	},
```
[the remark this answers is not in the document]
Comment (greatbacon) [reply src/lib/roadmap/roadmapData.ts:46]: 🤷

File: src/routes/roadmap/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+			class="roadmap-icon"
+			class:completed={item.status === 'completed'}
+			class:milestone={item.status === 'milestone'}
+			class:in-progress={item.status === 'in-progress'}
+			class:planned={item.status === 'planned'}>{item.icon}</span
+		>
+
+		<div>
+			<div class="title">{item.title}</div>
+			<div>{item.summary}</div>
```
Comment (ryanatkn) [inline src/routes/roadmap/+page.svelte:36]:
```suggestion
			<div>{@html item.summary}</div>
```

File: src/routes/roadmap/roadmapData.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+		title: 'First mockup published',
+		summary: 'A first mockup of the UX is put together and released.',
+		date: 'Q1 2020',
+	},
+	{
+		icon: '📝',
+		status: 'completed',
+		title: 'Published "Why Svelte"',
+		summary:
+			'Published this piece (https://github.com/feltcoop/why-svelte) about why we are using Svelte.',
```
Comment (ryanatkn) [inline src/routes/roadmap/roadmapData.ts:37]:
```suggestion
			'Published the technical article <a href="https://github.com/feltcoop/why-svelte">"Why Svelte is our choice for a large web project in 2020"</a>',
```

formatting may be screwed up by this change

**Review by ryanatkn — APPROVED** [review]

**Pull request merged**